libs/algorithms/include/hpx/parallel/algorithms/for_each.hpp

```diff
@@ -26,6 +26,7 @@
 #include <hpx/execution/algorithms/detail/is_negative.hpp>
 #include <hpx/executors/execution_policy.hpp>
 #include <hpx/parallel/algorithms/detail/dispatch.hpp>
```

Comment:
Is this needed?

Reply:
I think it's needed. This is the base class of the algorithm implementation (see `detail::for_each` below)